### PR TITLE
#3331  Implement zoom in/out and fit feature

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3331)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/App.js [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 import React from 'react';
 import styled from 'styled-components';
+import CssBaseline from '@material-ui/core/CssBaseline';
```
Comment (konekoya) [inline ohara-manager/client/src/App.js]: This makes Mui happy and includes some default styles for our App...

File: ohara-manager/client/src/components/Pipeline/Graph/Connector/ConnectorGraph.js [hunk 2]
```diff
@@ -55,7 +55,7 @@ const ConnectorGraph = params => {
       `<div class="type">${type === 'Pipeline Only' ? 'Topic' : type}</div>`,
       `<div class="status">`,
       `<div class="left">${'Status'}</div>`,
-      `<div class="right">${'pending'}</div>`,
+      `<div class="right">${'stopped'}</div>`,
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/Graph/Connector/ConnectorGraph.js]: Default state should be _stopped_

File: ohara-manager/client/src/components/Pipeline/Pipeline.js [hunk 3]
```diff
@@ -77,6 +68,8 @@ const Pipeline = () => {
     if (current === undefined && hasWorkspace) {
       history.push(`/${workspaces[0].settings.name}`);
     }
+  } else if (hasWorkspace) {
+    history.push(`/${workspaces[0].settings.name}`);
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/Pipeline.js]: Choose a default workspace when none is specified

File: ohara-manager/client/src/components/Pipeline/Pipeline.js [hunk 4]
```diff
@@ -130,29 +123,22 @@ const Pipeline = () => {
...
-                  handleToolboxClose={() => {
-                    setIsToolboxOpen(false);
-                    setToolboxExpanded(initialState);
-                  }}
-                  toolboxKey={toolboxKey}
-                  setToolboxExpanded={setToolboxExpanded}
-                />
-              </GraphWrapper>
-            </>
+            <Graph
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/Pipeline.js]: Lifting the state up so Toolbar component is able to consume states from Graph component

File: ohara-manager/client/src/App.js (hunk 1 above)
Comment (eechih) [reply ohara-manager/client/src/App.js]: Nice

File: ohara-manager/client/src/components/DevTool/Header/Header.js [hunk 5]
```diff
@@ -69,7 +69,7 @@ const DialogSelect = props => {
     disabled = false,
   } = props;
   return (
-    <Typography hidden={index !== currentTab}>
+    <Typography component="div" hidden={index !== currentTab}>
```
Comment (saivirtue) [inline ohara-manager/client/src/components/DevTool/Header/Header.js]: Thanks for helping this 😭

Comment (konekoya) [reply ohara-manager/client/src/components/DevTool/Header/Header.js]: No worried. It's tricky actually :)

**Pull request merged**